json_schema/project.schema.json

```diff
@@ -0,0 +1,310 @@
+{
+  "$id": "project.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "MODIP Metadata Format - Project",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "author": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "id",
+        "name"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "A unique identifier of this author."
+        },
+        "name": {
+          "type": "string",
+          "description": "A human-readable name of this author"
+        },
+        "uri": {
+          "type": "string",
+          "description": "A URI users can visit to view more about this author.",
+          "format": "uri"
+        }
+      }
+    }
+  },
+  "required": [
+    "schemaVersion",
+    "id",
+    "name"
+  ],
+  "properties": {
+    "$schema": true,
+    "schemaVersion": {
```

Comment:
I would use SemVer here, together with pattern matching:


```json
{
  "type":        "string",
  "description": "The version of the MODIP Metadata Format.",
  "pattern":     "^[0-9]+(\\.[0-9]+){0,2}$"
}
```

Reply:
Sounds good, I will change it over to SemVer.